pss/trojan/message.go

```diff
@@ -40,6 +40,11 @@ type Message struct {
 	padding []byte
 }
 
+// GetTopic exposes the underlying topic of a Message type
```

Comment:
if we really want the trojan.Message back we should export its fields, but i would probably just return the msg payload

Reply:
removed `GetTopic` and exporter `Topic` field instead